test/rai/pipeline_boston_analyse.yaml

```diff
@@ -87,3 +87,17 @@ jobs:
       insight_2: ${{parent.jobs.counterfactual_01.outputs.counterfactual}}
       insight_3: ${{parent.jobs.error_analysis_01.outputs.error_analysis}}
       insight_4: ${{parent.jobs.explain_01.outputs.explanation}}
+
+  scorecard_01:
```

Comment:
This needs to be hooked into one of the Python test files to get it to run. e.g.:
https://github.com/Azure/RAI-vNext-Preview/blob/a65b46830641931b897ead91d0d3f2e1b225d121/test/rai/test_rai_pipeline.py#L23

Reply:
Sorry my mistake. Wasn't looking closely enough and thought this was an add not an edit.

Reply:
synced with Richard, we agreed here.